backend/js/src/main/scala/aqua/backend/js/JavaScriptFunc.scala

```diff
@@ -113,12 +116,9 @@ case class JavaScriptFunc(func: FuncCallable) {
 object JavaScriptFunc {
 
   def argsToTs(at: ArrowType): String =
-    at.args.zipWithIndex
-      .map(_.swap)
-      .map(kv => "arg" + kv._1)
-      .mkString(", ")
+    at.domain.toLabelledList().map(_._1).mkString(", ")
```

Comment:
maybe use some helpers here to hide `domain` stuff. We already have `args` and `res` helper functions